Bookish/Models/EditBooks.cs

```diff
@@ -2,49 +2,41 @@
 
 public class EditBooks
 {
-    public static void InsertBooks(string author, string title)
+    static LibraryContext context = new LibraryContext();
+    public static List<BookModel> GetBookList()

```

Comment:
General comment: I love that you've moved logic out into a separate class, it's much easier to read! However, it probably doesn't belong in the `Models` folder. Instead, you could call this class and file something like `EditBooksHelper` and put it in a new folder called `Helpers`